panct/__main__.py

```diff
@@ -1,25 +1,63 @@
 #!/usr/bin/env python
+
 import typer
+from typing_extensions import Annotated
+from .complexity import AVAILALBE_METRICS
 
 app = typer.Typer()
 
 
 @app.command()
-def complexity(gfa_file: str, output_file: str):
+def complexity(
+    output_file: Annotated[str, typer.Option("--out", help="Name of output file")],
+    gfa_file: Annotated[
+        str, typer.Option("--gfa", help="Path to .gfa file of the graph")
+    ] = "",
+    gbz_file: Annotated[
+        str, typer.Option("--gbz", help="Path to .gbz file of the graph")
+    ] = "",
+    region: Annotated[
+        str, typer.Option("--region", help="Region to compute complexity over")
+    ] = "",
+    region_file: Annotated[
+        str,
+        typer.Option(
+            "--region-file", help="Bed file of regions to compute complexity over"
+        ),
+    ] = "",
+    metrics: Annotated[
+        str,
+        typer.Option(
+            "--metrics",
+            help="Comma-separated list of which "
+            "complexity metrics to compute. "
+            "Options: " + ",".join(AVAILALBE_METRICS),
```

Comment:
if you want, you can even have Typer check that the user input is one or multiple of the `AVAILABLE_METRICS` by typing this option as an Enum:

https://typer.tiangolo.com/tutorial/parameter-types/enum

Reply:
thanks for pointing out these options! skipping for now but once our interface is more finalized will revisit